### PR TITLE
Gitlab Mr109 - Add big karp tiny rat pokemon filtering (Credit: @versx)

### PR DESCRIPTION
**Pull request by whitewillem**

### CODE REVIEW

**Comment by whitewillem** [comment]
left 1 linting error. Rename it to whatever you think is best @123FLO321

File: resources/webroot/static/data/en.json [hunk 1]
```diff
@@ -3542,5 +3542,7 @@
         "filter_spawnpoint_options": "Options",
         "filter_spawnpoint_with_timer": "With Timer",
         "filter_spawnpoint_without_timer": "Without Timer"
```
Comment (SkOODaT) [inline resources/webroot/static/data/en.json:3544]: missing a `,` here for next line

**Comment by maglat** [comment]
Is the filter already in? Would love to have this

**Comment by Fabio1988** [comment]
Is this already merged to master? What is missing?

**Review by SkOODaT — APPROVED** [review]

**Comment by Fabio1988** [comment]
Too outdated, sorry. Not sure if anyone wants to rework it on latest development, feel free. I will merge it then.

**Pull request closed without merging**